src/Tests/Consolonia.TestsCore/Consolonia.TestsCore.csproj

```diff
@@ -1,9 +1,15 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
     <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Core.Build.props', '$(MSBuildThisFileDirectory)../'))" />
-    
+
+    <PropertyGroup>
+        <IsPackable>false</IsPackable>
```

Comment:
I thought lets publish TestsCore to make testing available to users. But now the meaning of this assembly is different, it's tests itself now.

Reply:
OH. I see. It felt weird to put Pixel tests into a gallery test assembly.  Should I create a new Core.Tests assembly?